addons/pos_self_order/static/src/app/self_order_index.js

```diff
@@ -37,6 +37,15 @@ export class selfOrderIndex extends Component {
 
     setup() {
         this.selfOrder = useSelfOrder();
+        onWillStart(async () => {
+            if ("serviceWorker" in navigator) {
+                navigator.serviceWorker
```

Comment:
```suggestion
                await navigator.serviceWorker
```

Reply:
I'm pratically sure that the await do not change anything here :/ It do not return any promise. 
There is one promise in the ready `navigator.serviceWorker.ready` but I think awaited this can block the whole UI.